### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by MarkusL96** [aumath-advancedr2019/Permutation1#20]
### 1. Purpose of the package

Description file may contain the maintainers (and authors) and a short description like in the vignette where you have described the package very well in a compact way. Anyway, taking the vignette into account the purpose of the package is clear.

### 2. Completeness

Since there is no data included to your package I would suggest that you include the test data that you have used in your vignette in case you want to include some data.

### 3. Code quality and sophistication

The package installation was straight-forward and I did not encounter any problems at all.

Besides your examples, your functions also run perfectly well for examples I made up myself.

You have also used advanced programming techniques (using the existing class "htest") in a very appropriate way, which is great. 

### 4. Documentation

All functions are very well documented including useful examples.

Your vignette is very helpful since it provides the necessary background to understand roughly the underlying mathematical background. Furthermore, it shows examples for all functions which make it easier to understand how to use the functions.

Some further remarks concerning your vignette:

In the subsection "Using permutation_mean()" of your vignette you refer to the p-value of your example. As far as I understood, it is given by 0.0108. Nevertheless, you have written "Here we get a p-value of $3∗10−4$, and with a 5% significance level we can reject that the mean of the two groups is the same.", which is possibly a typo. One possible way to avoid this is to directly refer in your markdown file to the p-value via "\`r p$p.value`".
I think the same applies to the second example.

### 5. Interface

The user interface feels very natural and is in line what I am used to from other packages.

I also appreciate that you've included warnings and error messages helping the user working with your functions. In my opinion, your error messages and warnings are intelligible.

In addition, I liked the option in your functions to use ggplot and get more sophisticated plots.

### 6. Conclusion

In summary, I liked your package. In my opinion, it has met all the requirements (like S3 objects, nice plots, self-explanatory vignette, ..) for a good package. Apart from my minor remarks, I think nothing needs to be improved :)

**Comment by SolveigOe** [comment]
Thank you for the nice feedback :)
